engine/graphic/src/Texture.cpp

```diff
@@ -0,0 +1,86 @@
+#include "Texture.h"
+Texture::Texture(char* FileName)
+{
+	m_fileName = FileName;
+	imageId = 0;
```

Comment:
ImageId is being set this way:
13 ilGenImages(1, &imageId);
value before it doesent matter, its just an initialization